score-rce/src/main/java/io/cloudslang/content/remote/utils/StringUtils.java

```diff
@@ -0,0 +1,20 @@
+package io.cloudslang.content.remote.utils;
```

Comment:
@dashapaster You should avoid StringUtils as name for a class because the Apache.Commons.Lang has one identical... so is misleading...